api/TrackDefault.json

```diff
@@ -34,6 +34,9 @@
           "safari_ios": {
             "version_added": false
           },
+          "samsunginternet_android": {
+            "version_added": null
```

Comment:
This file is probably fine since I don't think there is an implementation of this in any browser.